### DISCUSSION TITLE
Best way to display the IP address on ssd1306 on first boot after flash?

### DISCUSSION THREAD

**Discussion by clementpeleman** [arendst/Tasmota#17757] [Q&A]
What would be the best method to display the IP address of the tasmota device when booting for the first time (with ssid and pass hardcoded in the bin file). I can now do this with the displaymode 2 command but this requires me to know the IP address in advance. 

Thanks!
Clement Peleman

**Comment by sfromis** [comment]
Since you create a bin with the credentials, you could also include rules for doing such things.
https://tasmota.github.io/docs/Compile-your-build/#advanced-customization

Of course, you'd also need to include a template to have the I2C pins for the SSD1306 configured.
